src/go/rpk/pkg/cli/redpanda/admin/brokers/decommission-status.go

```diff
@@ -51,6 +52,12 @@ func newDecommissionBrokerStatus(fs afero.Fs, p *config.Params) *cobra.Command {
 				}
 			}
 
+			if dbs.AllocationFailures != nil {
+				out.Section("allocation failures")
+				fmt.Printf("%s\n\n", dbs.AllocationFailures)
```

Comment:
Can we
```
sort.Strings(dbs.AllocationFailures)
for _, f := range dbs.AllocationFailures {
  fmt.Println(f)
}
```
?

Reply:
Also we might want to explain allocation failures in the long help text

Reply:
Although we don't have any long help text right now, it appears.

Reply:
Will add help text too.

Reply:
@twmb , does this long help text look good?
```
When a node is in decommissioning, it prints the progress below by default.

DECOMMISSION PROGRESS
=====================
$ rpk redpanda admin brokers decommission-status 4
NAMESPACE-TOPIC              PARTITION  MOVING-TO  COMPLETION-%  PARTITION-SIZE
kafka/test                   0          3          9             1699470920
kafka/test                   4          3          0             1614258779
kafka/test2                  3          3          3             2722706514
kafka/test2                  4          3          4             2945518089
kafka_internal/id_allocator  0          3          0             3562

With '--detailed' or '-d', it prints granular outputs like below.

DECOMMISSION PROGRESS
=====================
$ rpk redpanda admin brokers decommission-status 4 -d
NAMESPACE-TOPIC  PARTITION  MOVING-TO  COMPLETION-%  PARTITION-SIZE  BYTES-MOVED  BYTES-REMAINING
kafka/test       0          3          13            1731773243      228114727    1503658516
kafka/test       4          3          1             1645952961      18752660     1627200301
kafka/test2      3          3          5             2752632301      140975805    2611656496
kafka/test2      4          3          6             2975443783      181581219    2793862564

If a partition cannot be moved with some reason, it will be reported in the 'ALLOCATION FAILURES' section and decommission never succeeds. Note that ALLOCATION FAILURES is reported only when 'partition_autobalancing_mode' is set to 'continuous'.

ALLOCATION FAILURES
==================
kafka/foo/2
kafka/test/3
```